domain-models-runtime/src/main/java/org/folio/rest/RestRouting.java

```diff
@@ -504,18 +505,15 @@ private static void handleStream(Method method2Run, RoutingContext rc, Object in
     });
   }
 
-  static void getOkapiHeaders(RoutingContext rc, Map<String, String> headers, String[] tenantId) {
-    MultiMap mm = rc.request().headers();
+  static String getOkapiHeaders(MultiMap mm, Map<String, String> headers) {
```

Comment:
static CaseInsensitiveMap<String, String> getOkapiHeaders(MultiMap mm) {

and

    Map<String, String> okapiHeaders = getOkapiHeaders(rc.request().headers());
    String tenantId = okapiHeaders.get(XOkapiHeaders.TENANT);

is easier to understand.

Reply:
agreed.